src/coreclr/scripts/superpmi.py

```diff
@@ -1495,6 +1495,7 @@ def replay_with_asm_diffs(self):
             "DOTNET_JitDisasm": "*",
             "DOTNET_JitUnwindDump": "*",
             "DOTNET_JitEHDump": "*",
+            "DOTNET_JitAlignLoops": "0", # disable loop alignment to filter noise
```

Comment:
Is there any way we can also include a section that shows loop alignment diffs?

It is sometimes interesting to see where a given change significantly impacts alignment of methods so we can preliminarily see potential impact.

Reply:
Basically, I think it would be great long term if we could both see "diff without loop alignment" and "diff with loop alignment".

That way we can better judge "this isolated change is an improvement vs regression" while also taking into account the real world impact on assembly size.

Reply:
Getting both numbers would require running diffs twice. That seems like overkill for every change.

Reply:
It also seems like code / assembly size isn't a real concern here, since the code being generated doesn't have any influence over the alignment padding added.